apps/wallet/src/ui/app/hooks/deepbook/useGetEstimate.ts

```diff
@@ -117,8 +114,9 @@ async function getPlaceMarketOrderTxn({
 
 	if (isAsk) {
 		const bigNumberBaseBalance = new BigNumber(baseBalance);
+		const oneSuiDeepBook = new BigNumber(1).shiftedBy(USDC_CONVERSION_RATE);
 
-		if (bigNumberBaseBalance.isLessThan(ONE_SUI_DEEPBOOK)) {
+		if (bigNumberBaseBalance.isLessThan(oneSuiDeepBook)) {
```

Comment:
I've removed this calculation for less than 1 SUI. Tested and it does go through correctly

Reply:
Yeah it will probably only fail when the swap max is selected

Reply:
@pchrysochoidis I updated the code so that when max is selected, it will take off some values so that it can swap correctly. Otherwise, it will use the same amount and sweep over.